lib/parser.js

```diff
@@ -23,7 +23,7 @@ function merge(query, next) {
 }
 
 function parseFilter (filter) {
-	let filterConf = /^([^\!\~\?]*)(\!\?|\!=|=|>|<|\~|\?)([^=\~\?]*)$/.exec(filter);
+	let filterConf = /^([^\!\~\?]*)(\!\?|\!=|=|>|<|\~|\!\~|\?)([^=\~\?]*)$/.exec(filter);
```

Comment:
could you change this to

```
/^([^\!\~\?]*)(\!\?|\!=\!\~||=|>|<|\~|\?)([^=\~\?]*)$/
```

I may be being overly cautious, but I can never remember how greedy various regex matchers are, so am putting the 2-character ones at the beginning